rules_building_block/defense_evasion_creation_of_hidden_files_directories.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2023/08/23"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/08/23"
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+Identify activity related where adversaries can include a leading . character or add the 'hidden' flag to files to hide
+them from the user in an attempt to evade detection
+"""
+from = "now-119m"
+interval = "60m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Hidden Files and Directories via Leading Character or Hidden Flag"
+risk_score = 21
+rule_id = "5124e65f-df97-4471-8dcb-8e3953b3ea97"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "OS: macOS","Use Case: Threat Detection", "Tactic: Defense Evasion", "Rule Type: BBR"]
+timestamp_override = "event.ingested"
+type = "eql"
+query = '''
+file where event.type : "creation" and process.name : "chflags"
```

Comment:
We already have the covered in rules/linux/defense_evasion_hidden_file_dir_tmp.toml. This specifically works on suspicious directory path. Having this work on all paths is extremely noisy that can create alerts on all known things dev like ~/.bash_history , ~/.creds, ~/.ssh and others. So since we have the coverage in prod, I just included chflags for this BBR

Reply:
should you change the description to match then?

Reply:
Done.